web/src/components/Footer.tsx

```diff
@@ -34,27 +40,23 @@ const FooterContainer = styled.footer`
  * The standard footer which can supplied to a Layout. Displays a list of links from the props and adds the version
  * number if it's a dev build.
  */
-class Footer extends React.PureComponent<PropsType> {
-  static getVersion(): ReactNode {
-    if (buildConfig().featureFlags.developerFriendly) {
-      return (
-        <span>
-          {__VERSION_NAME__}+{__COMMIT_SHA__}
-        </span>
-      )
-    }
-    return null
-  }
 
-  render(): ReactNode {
-    const { children } = this.props
+const getVersion = (): ReactNode => {
+  if (buildConfig().featureFlags.developerFriendly) {
     return (
-      <FooterContainer>
-        {children}
-        {Footer.getVersion()}
-      </FooterContainer>
+      <span>
+        {__VERSION_NAME__}+{__COMMIT_SHA__}
+      </span>
     )
   }
+  return null
 }
```

Comment:
Why did you extract this? Please inline :)

Reply:
moved inline